### ISSUE TITLE
Not working behind authenticated proxy

### ISSUE THREAD

**Issue by phw198** [phw198/OutlookGoogleCalendarSync#4]
Keep getting 407 error even though Proxy setting has been configured with
username and password.  
Can anyone give me a light on this issue?  

Work Item Details
--------------------
**Original CodePlex Issue:**	[Issue 137](http://outlookgooglecalendarsync.codeplex.com/workitem/137)
**Status:**	Proposed
**Reason Closed:**	Unassigned
**Assigned to:**	Unassigned
**Reported on:**	Dec 10, 2015 at 1:16 PM
**Reported by:**	Dmlt12y
**Updated on:**	Dec 31, 2015 at 1:56 AM
**Updated by:**	Dmlt12y

**Comment by phw198** [comment]
On *2015-12-28 00:45:12 UTC*, **phw198** commented:

There are so many different proxies that users could be accessing through, this is always hard to troubleshoot. Generally it's best to make sure you can get to the internet with Internet Explorer, and then let the sync tool inherit its settings from IE.
 Is this not an option for you?

**Comment by phw198** [comment]
On *2015-12-28 09:34:42 UTC*, **Dmlt12y** commented:

@phw198 appreciate you response!
The internet works fine with internet explorer. Tried to let the tool inherit setting from IE but 407 error still haunts me. One thing I found out is that with or without username and password for proxy authentication, I always get this 407 error. This leads
 me to a speculation that the authentication support for proxy might not be implemented correctly.
